new_client/src/App.js

```diff
@@ -1,11 +1,21 @@
 import React from 'react';
-import Auth from "./auth/Auth";
+import './App.css';
+
+// Imports of pages
+import Header from "./components/Header.js"
+import Home from './components/Home/home.js';
+import Footer from './components/Footer.js'
+
+
 function App() {
   return (
     <div className="App">
-      <Auth/>
```

Comment:
You removed the Auth component!  
Its okay for now  
@ShubhankarKG @pal-16 please reinclude it in the next commit.

Also, the package-lock.json may also be a bit broken I feel